### DISCUSSION TITLE
Sherlock-project

### DISCUSSION THREAD

**Discussion by Henry6661997** [sherlock-project/sherlock#895] [Q&A] [answered]
When I get to the requirements part it saying  /usr/bin/python3: No module named pip. Im new to kali linux and im not sure what to do.

**Comment by hoadlck** [comment, marked as answer]
As the error says, you do not have pip installed on your machine.  Google for installing pip on Linux, and you will see many examples.

This is not an issue with Sherlock itself: it is a problem with your environment.
